modules/light-clients/08-wasm/types/client_state_test.go

```diff
@@ -365,7 +384,7 @@ func (suite *TypesTestSuite) TestVerifyMembershipGrandpa() {
 						ConnectionId: connectionID,
 						Prefix:       suite.chainA.GetPrefix(),
 					},
-					DelayPeriod: 0,
+					DelayPeriod: 1000000000,
```

Comment:
Curious: why this needed to be changed?

Reply:
Since the last test data was generated, a requirement was added to Hyperspace for a non-zero delay in seconds. For generating the new test data using the e2e test, I used 1 second, 1_000_000_000, for that input to fulfill that requirement.

Reply:
Thanks for the explanation, @misko9. I will add a comment in a follow-up PR, so that we don't lose this information.